http-easy-lib/http-easy/private/pool.rkt

```diff
@@ -1,8 +1,8 @@
 #lang racket/base
 
-(require (prefix-in d: data/pool)
-         net/http-client
+(require net/http-client
          racket/contract/base
+         (prefix-in d: data/pool)
```

Comment:
Gonna skip this one as discussed on Discord and in https://github.com/jackfirth/resyntax/issues/432 .